.github/workflows/build-images.yml

```
@@ -204,7 +204,7 @@ jobs:
       pull-request-target: "true"
       is-committer-build: ${{ needs.build-info.outputs.is-committer-build }}
       push-image: "true"
-      use-uv: ${{ needs.build-info.outputs.force-pip && 'false' || 'true' }}
+      use-uv: ${{ needs.build-info.outputs.force-pip=='true' && 'false' || 'true' }}
```

```suggestion
      use-uv: ${{ needs.build-info.outputs.force-pip == 'true' && 'false' || 'true' }}
```

nits: spaces seem to be used elsewhere 👀